packages/validator/test/capability.spec.js

```diff
@@ -1503,10 +1506,10 @@ test('capability with optional caveats', async () => {
   const Echo = capability({
     can: 'test/echo',
     with: URI.match({ protocol: 'did:' }),
-    nb: {
+    nb: Schema.struct({
```

Comment:
wdyt of trying to make things backward incompatible so code like `nb: { message: URI }` still works even without wrapping with `Schema.struct`?

Reply:
I wanted to, but amount of added complexity coupled with the fact that TS inference gets confused on occasions I don’t think it would be worth it